controllers/users.mjs

```diff
@@ -0,0 +1,127 @@
+import { db } from "../firebase.mjs";
+
+// Firestore collection name
+const collectionName = "users";
```

Comment:
Maybe "usersCollection"?

Just a suggestion for semantically knowing what "collectionName" is retrieving from the the various methods below, without returning above.